<launch>
	<arg name="master_start_pos" default="0 0 0"/>
	<arg name="master_goal_pos" default="0 2 0"/>
	<arg name="follower_start_pos" default="-1 0 0"/>
	<arg name="follower_goal_pos" default="0 -2 0"/>
	<arg name="min_drop_dist" default="1.0"/>

	<node name="path_planner" type="path_planner.py" pkg="path_planner">
		<param name="master_start_pos" value="$(arg master_start_pos)"/>
		<param name="master_goal_pos" value="$(arg master_goal_pos)"/>
		<param name="follower_start_pos" value="$(arg follower_start_pos)"/>
		<param name="follower_goal_pos" value="$(arg follower_goal_pos)"/>
		<param name="min_drop_dist" value="$(arg min_drop_dist)"/>
	</node>
</launch>